#!/usr/bin/env python
PACKAGE = "hydra_controllers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("translation_gain", double_t, 0, "Cartesian translation gain", 100, 0, 500)
gen.add("rotation_gain", double_t, 0, "Cartesian rotation gain", 30, 0, 500)
gen.add("redundancy_gain", double_t, 0, "Redundancy manipulability gain", 500, 0, 500)
gen.add("positioner_gain", double_t, 0, "Positioner gain", 5, 0, 500)

exit(gen.generate(PACKAGE, "dynamic_taskpriorityl", "hydra_param"))